Easy/4. Digital Hospital Manager/solutions/pallavisavant_Pallavi Savant/patients_view.php

```diff
@@ -15,10 +15,14 @@
  	echo "<td>" .$row['Weight']. "</td>";
  	echo "<td>" .$row['Height']. "</td>";
  	echo "<td>" .$row['Symptoms']. "</td>";
- 	echo "<td>" .$row['Date']. "</td>";
+ 	echo "<td>" .$row['Date Of Admission']. "</td>";
```

Comment:
There seems to an error because of this change. Refer Screenshots.
![Screenshot 2020-10-20 at 1 16 54 PM](https://user-images.githubusercontent.com/54791364/96556707-f48b4000-12d6-11eb-9218-42cba356c829.png)

Reply:
@girlscript-blr fixed the bug...Thank you:)